README.md

```diff
@@ -35,6 +35,11 @@ sudo apt-get install git
 sudo apt-get install openjdk-7-jdk
 sudo apt-get install npm
 sudo apt-get install libfontconfig
+sudo npm -g install grunt-cli
```

Comment:
Unfortunately, this was not my experience.  I followed these instructions on a fresh CentOS 7 install and the zeppelin-web module build failed without the npm install steps.  Are you saying that it should work if you remove the `sudo apt-get install npm` step?